bee-inx/src/treasury.rs

```diff
@@ -25,6 +25,15 @@ impl TryFrom<proto::TreasuryOutput> for TreasuryOutput {
     }
 }
 
+impl From<TreasuryOutput> for proto::TreasuryOutput {
+    fn from(value: TreasuryOutput) -> Self {
+        Self {
+            milestone_id: Some(value.milestone_id.into()),
```

Comment:
Just curious, why is this an Option?

Reply:
Because everything except for primitive types in `protobuf`s are optional:

https://docs.rs/inx/1.0.0-beta.8/inx/proto/struct.TreasuryOutput.html